package-lock.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
I think this file was accidentally deleted

Reply:
Don't think the other plugin-samples have `package-lock.json`

Reply:
oh wait this isn't in the trivia folder